src/picknik_ur_site_config/objectives/open_door_lever_handle_ml.xml

```diff
@@ -1,26 +1,25 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <root BTCPP_format="4" main_tree_to_execute="Open Door Lever Handle ML">
   <!-- ////////// -->
   <BehaviorTree ID="Open Door Lever Handle ML" _description="Attempts to open a door with a lever handle using machine learning." _favorite="false" _hardcoded="true">
     <Control ID="Sequence" name="TopLevelSequence">
       <!-- Load the parameters needed for this Objective -->
-      <Action ID="LoadObjectiveParameters" config_file_name="open_door_lever_handle_MTC_config.yaml" parameters="{parameters}"/>
       <!-- Update the admittance controller and reset the FTS -->
+      <!-- Look at the desired location where objects (door) will be present -->
+      <Action ID="LoadObjectiveParameters" config_file_name="open_door_lever_handle_MTC_config.yaml" parameters="{parameters}"/>
       <SubTree ID="Re-Zero Force-Torque Sensors" _collapsed="true"/>
       <SubTree ID="Update Admittance Controller" _collapsed="true"/>
       <!-- Look at the desired location where objects (door) will be present -->
```

Comment:
this comment appears twice on lines 8 and 12, probably from auto-saving. Just keep one in the right place